release/go.mod

```diff
@@ -8,67 +8,160 @@ require (
 	github.com/aws/eks-distro-build-tooling/release v0.0.0-20211103003257-a7e2379eae5e
 	github.com/fsouza/go-dockerclient v1.7.2
 	github.com/ghodss/yaml v1.0.0
+	github.com/go-logr/logr v1.2.0
```

Comment:
Can you try running `go mod tidy` in the `release` directory to make sure all these dependencies are actually needed?

Reply:
Also looks like some of the dependencies versions have been bumped up. Are we sure we need that?

Reply:
I was using logr for the helm driver, since it's the same pattern we use for package controller in there.

Reply:
Can you try running `go mod tidy` in the `release` directory? It has some go.mod file changes. Other than this everything else looks good.